added instructor note about environment pane (#843)
datacarpentry/R-ecology-lesson: episodes/02-starting-with-data.Rmd

content and the structure of a data frame. Here is a non-exhaustive list of
functions to get a sense of the content/structure of the data. Let's try them out!

::: instructor

> As you go through this next section, you may want to point out what is happening in the enviornement pane. Specially with the function 'nrow' and 'ncol' as they correspond with 'observation' and 'variables' in the enviornment. This can help learners make connections between the different sections of the R interface and identify ways to double check their work. 

Reviewer: fix typo ("environment"). and maybe instead of the sentence "Specially with...", something like "For example, the functions 'now' and 'col' correspond with 'observation' and 'variables' in the environment pane."

> As you go through this next section, you may want to point out what is happening in the enviornement pane. Specially with the function 'nrow' and 'ncol' as they correspond with 'observation' and 'variables' in the enviornment. This can help learners make connections between the different sections of the R interface and identify ways to double check their work. 

Reviewer: ```suggestion
As you go through this next section, you may want to point out what is happening in the environment pane. Specially with the function 'nrow' and 'ncol' as they correspond with 'observation' and 'variables' in the enviornment. This can help learners make connections between the different sections of the R interface and identify ways to double check their work. 
```

> As you go through this next section, you may want to point out what is happening in the enviornement pane. Specially with the function 'nrow' and 'ncol' as they correspond with 'observation' and 'variables' in the enviornment. This can help learners make connections between the different sections of the R interface and identify ways to double check their work. 

Reviewer: ```suggestion
As you go through this next section, you may want to point out what is happening in the enviornement pane. For example, the functions 'now' and 'col' correspond with 'observation' and 'variables' in the environment pane. This can help learners make connections between the different sections of the R interface and identify ways to double check their work. 
```

Reviewer 2: @joongsup the following line "the functions 'now' and 'col'" should be 

"the functions `nrow()` and `ncol()`". 

I am unable to edit your suggestion, so could you update it? Thanks.

> As you go through this next section, you may want to point out what is happening in the enviornement pane. Specially with the function 'nrow' and 'ncol' as they correspond with 'observation' and 'variables' in the enviornment. This can help learners make connections between the different sections of the R interface and identify ways to double check their work. 

Reviewer: ```suggestion
As you go through this next section, you may want to point out what is happening in the enviornement pane. For example, the functions `nrow()` and `ncol()` correspond with 'observation' and 'variables' in the environment pane. This can help learners make connections between the different sections of the R interface and identify ways to double check their work. This can help learners make connections between the different sections of the R interface and identify ways to double check their work. 
```

Reviewer 2: All good! Thanks. Let's wait to hear from @jwinston00 and/or @sstevens2.
